examples/search_examples/server/my_strategy.ts

```diff
@@ -29,3 +32,54 @@ export const mySearchStrategyProvider = (
     },
   };
 };
+
+type FibonacciRequest = IKibanaSearchRequest<{ n: number }>;
+
+type FibonacciResponse = IKibanaSearchResponse<{ values: number[] }>;
+
+export const fibonacciStrategyProvider = (): ISearchStrategy<
```

Comment:
Could we place these in two files?
As well as putting all the types into the `types` file?